R/process_data.R

```diff
@@ -532,6 +532,14 @@ process_production_data <- function(data, start_year, end_year, time_horizon,
         .data$plan_emission_factor
       )
     )
+  data_processed <- data_processed %>%
+    dplyr::mutate(
+      plan_sec_prod = dplyr::if_else(
+        is.na(.data$plan_sec_prod),
+        0,
+        .data$plan_sec_prod
+      )
+    )
```

Comment:
this has been removed but the wrangle_and_check_production_data() function is still in effect. We should remove it as what it does is filling missing year value, and we expect that there are none. Could be replaced by an assertr::verify operation to check this assumption ?